build.sbt

```diff
@@ -49,30 +50,46 @@ name := "cloudstate"
 
 val GrpcJavaVersion = "1.22.1"
 val GraalAkkaVersion = "0.5.0"
-val AkkaVersion = "2.5.31"
+val AkkaVersion = "2.6.5"
 val AkkaHttpVersion = "10.1.11"
 val AkkaManagementVersion = "1.0.5"
 val AkkaPersistenceCassandraVersion = "0.102"
 val PrometheusClientVersion = "0.6.0"
 val ScalaTestVersion = "3.0.5"
-val ProtobufVersion = "3.9.0"
+val ProtobufVersion = "3.9.0" // We use this version because it is the latest which works with native-image 20.0.0
 val GraalVersion = "20.0.0"
```

Comment:
20.0.1?

Reply:
@hepin1989 2.1.0 will be released in 2 weeks, so holding off making modifications to GraalVM until then.